### DISCUSSION TITLE
upscaling questions

### DISCUSSION THREAD

**Discussion by Sirmadsen** [Stability-AI/StableSwarmUI#403] [General]
Is there any way to choose upscale method and denoise strength? As it is now when making a 2x upscale it changes to much and sometime it looks really bad.

**Comment by TheLounger** [comment]
This is done through Refiner.
Refiner Control Percentage is denoising strength.
Upscale models go in `Models/upscale_models` directory, then you can select them under Refiner Upscale Method.

**Reply by Sirmadsen** [reply]
So if I don't want a refiner model but want to use the upscale variants I just activate refiner choose "Use Base" as the model?

And, does the refiner settings also apply to the upscale method that you can do on a image after it is created?

**Reply by mcmonkey4eva** [reply]
`So if I don't want a refiner model but want to use the upscale variants I just activate refiner choose "Use Base" as the model?`
Yes

`And, does the refiner settings also apply to the upscale method that you can do on a image after it is created?`
No, but you can "Use As Init" and then set init creativity to 0 at which point refiner is all that applies

**Reply by Sirmadsen** [reply]
Awesome, thanks.
